joern-cli/frontends/x2cpg/src/main/scala/io/joern/x2cpg/passes/frontend/PythonTypeHintCallLinker.scala

```diff
@@ -0,0 +1,44 @@
+package io.joern.x2cpg.passes.frontend
+
+import io.shiftleft.codepropertygraph.Cpg
+import io.shiftleft.codepropertygraph.generated.nodes.{Call, Method}
+import io.shiftleft.codepropertygraph.generated.{EdgeTypes, PropertyNames}
+import io.shiftleft.passes.CpgPass
+import io.shiftleft.semanticcpg.language._
+import overflowdb.traversal.Traversal
+
+/** Attempts to set the <code>methodFullName</code> and link to callees using the recovered type information from
+  * [[PythonTypeRecovery]]. Note that some methods may not be present as they could be external and have been
+  * dynamically discovered, thus the [[io.joern.x2cpg.passes.base.MethodStubCreator]] would have missed it.
+  *
+  * @param cpg
+  *   the target code property graph.
+  */
+class PythonTypeHintCallLinker(cpg: Cpg) extends CpgPass(cpg) {
+
+  implicit private val resolver: NoResolve.type = NoResolve
+
+  override def run(builder: DiffGraphBuilder): Unit = {
+
+    def calls: Traversal[Call] = cpg.call
+      .filterNot(c => c.name.matches("^(<operator>|import).*"))
```

Comment:
Would be good to comment here in which parts are language agnostic and which are language specific. I take it the `<operator>`-part is language agnostic, while the `import` is specific to the python frontend.